hack/update-crd-groups.sh

```diff
@@ -39,3 +39,15 @@ to-crdgen generate backupschedule >> $crd_target
 to-crdgen generate tidbmonitor >> $crd_target
 to-crdgen generate tidbinitializer >> $crd_target
 to-crdgen generate tidbclusterautoscaler >> $crd_target
+
+go install github.com/ahmetb/gen-crd-api-reference-docs
```

Comment:
install its binary release like `hack::ensure_kubectl`, then `go.(mod/sum)` will not be touched and `go mod tidy` is not required

Reply:
updated